Emeryherve/Free-Mentors
# chore(Database): create DB queries class file

#### What does this PR do?
- Create DataBase queries class file
- Add Postgres dependency for further use
- Add CRUD query operations in one file
#### Description of Task to be completed?
- This PR creates a new file that will be used by each needed CRUD operation and also these operations will need some additional dependencies that had also been added
#### How should this be manually tested?
-  We assume you have made a `clone` or [Download](https://github.com/Emeryherve/Free-Mentors/archive/ch-DB-modular-class-168330683.zip) of this project
1. Check in the root directory
2. Go to server
3. Navigate to models
4. Find a file called `dbQuery.js` 
#### Any background context you want to provide?
      N/A
#### What are the relevant pivotal tracker stories?
[PT Story](https://www.pivotaltracker.com/story/show/168330683)
#### Screenshots (if appropriate)
![imp1](https://user-images.githubusercontent.com/52406178/64389452-05c9ac80-d043-11e9-8ffe-ad7ff8fb3c88.JPG)
![imp2](https://user-images.githubusercontent.com/52406178/64389453-05c9ac80-d043-11e9-91d0-880a22132c10.JPG)
![imp3](https://user-images.githubusercontent.com/52406178/64389454-06624300-d043-11e9-8f1a-ba9d3aa69ed9.JPG)
![imp4](https://user-images.githubusercontent.com/52406178/64389455-06624300-d043-11e9-8368-dc4569218be2.JPG)
#### Questions:
N/A
